src/Test.UnitTests.Sarif/Readers/DelegatingStreamTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Readers;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Readers
+{
+    public class DelegatingStreamTests
```

Comment:
>DelegatingStreamTests [](http://example.com/codeflow?start=17&length=21)

Do you tests actually try to dispose the stream and ensure that it hasn't been disposed? That's the primary purpose of this wrapper, yes? So where's the test for that basic capability?

Reply:
just added.